### PR TITLE
feat: Add support for the Notifications API

### PR DESCRIPTION
**Pull request by elliotbruneel**
This MR allow to send notifications to authenticated user, projects members or organization members

Fix: https://github.com/crowdin/crowdin-api-client-php/issues/142

### CODE REVIEW

File: src/CrowdinApiClient/Api/Enterprise/NotificationApi.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    public function sendNotificationToOrganizationMembers(array $data)
+    {
+        $path = 'notify';
+
+        $options = [
+            'body' => json_encode($data),
+            'headers' => ['Content-Type' => 'application/json']
+        ];
+
+        return $this->client->apiRequest('post', $path, null, $options);
```
Comment (andrii-bodnar) [inline src/CrowdinApiClient/Api/Enterprise/NotificationApi.php:34]: Use the `_create` method

File: src/CrowdinApiClient/Api/NotificationApi.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    public function sendToAuthenticatedUser(array $data)
+    {
+        $path = 'notify';
+
+        $options = [
+            'body' => json_encode($data),
+            'headers' => ['Content-Type' => 'application/json']
+        ];
+
+        return $this->client->apiRequest('post', $path, null, $options);
```
Comment (andrii-bodnar) [inline src/CrowdinApiClient/Api/NotificationApi.php:29]: Use the `_create` method

File: src/CrowdinApiClient/Api/NotificationApi.php [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    public function sendNotificationToProjectMembers(int $projectId, array $data)
+    {
+        $path = sprintf('projects/%s/notify', $projectId);
+
+        $options = [
+            'body' => json_encode($data),
+            'headers' => ['Content-Type' => 'application/json']
+        ];
+
+        return $this->client->apiRequest('post', $path, null, $options);
```
Comment (andrii-bodnar) [inline src/CrowdinApiClient/Api/NotificationApi.php:53]: Use the `_create` method

File: src/CrowdinApiClient/Api/Enterprise/NotificationApi.php (hunk 1 above)
Comment (elliotbruneel) [reply src/CrowdinApiClient/Api/Enterprise/NotificationApi.php:34]: `_create` method need a model name. The api return an 204 code with no data.

File: src/CrowdinApiClient/Api/NotificationApi.php (hunk 2 above)
Comment (elliotbruneel) [reply src/CrowdinApiClient/Api/NotificationApi.php:29]: `_create` method need a model name. The api return an 204 code with no data.

File: src/CrowdinApiClient/Api/NotificationApi.php (hunk 3 above)
Comment (elliotbruneel) [reply src/CrowdinApiClient/Api/NotificationApi.php:53]: `_create` method need a model name. The api return an 204 code with no data.

**Review by andrii-bodnar — APPROVED** [review]
@elliotbruneel thank you for the contribution!

**Pull request merged**